action.yaml

```diff
@@ -17,15 +17,23 @@ inputs:
 runs:
   using: "composite"
   steps:
-    - name: Download Coder binary from access URL
-      run: curl -fsSL $CODER_URL/bin/coder-linux-amd64 -o /usr/local/bin/coder && chmod +x /usr/local/bin/coder
+    - name: Download and install Coder binary
+      run: |
+        if [ "${{ runner.arch }}" == "ARM64" ]; then
+          ARCH="arm64"
+        else
+          ARCH="amd64"
+        fi
+        mkdir -p $HOME/.local/bin
+        curl -fsSL ${{ inputs.access_url }}/bin/coder-linux-${ARCH} -o $HOME/.local/bin/coder
```

Comment:
Any reason to change this from env to template value? Env is safer with regards to shell escaping. I.e. `"${CODER_URL}"` will definitely work, whereas `"${{inputs.access_url}}"` could contain `"` and break the script.

Reply:
My thinking is that the ENV is not available at this stage so we should use the input directly.

Reply:
Hmm, why isn't it available if set via `env:` in yaml? If that's the case then sure.

Reply:
addressed.